CollectNewspaperKarel.java

```diff
@@ -13,16 +13,22 @@
 public class CollectNewspaperKarel extends SuperKarel {
 
 	// You fill in this part
-	// Added a comment
-	public void run(){
-		
+	private void goToDoor(){
 		move();
 		turnRight();
 		move();
 		turnLeft();
 		move();
 		move();
+	}
+	
+	private void collectNewsPeper(){
+		
 		pickBeeper();
+		
+	}
+	
+	private void turnBackHome(){
 		turnRight();
```

Comment:
for these two turnRight calls... Can you use a further decomposition to make it more meaningful? You have already studied that decomposition in cases